theme/templates/resource-landing-page/references.html

```diff
@@ -27,9 +27,7 @@ <h4>Related Resources</h4>
                     {% if relation.type|lower != "haspart" %}
                         <tr>
                             <td class="dataset-label">
-                                {% if relation.type == "isHostedBy" %}
-                                    The content of this resource is hosted by:
-                                {% elif relation.type == "isCopiedFrom" %}
+                                {% if relation.type == "isCopiedFrom" %}
```

Comment:
More relation code changes.  Why?

Reply:
Developers are perhaps most accustomed to having GitHub Issues to review for the functional requirements and use cases for a Pull Request. In this case, a google document has substituted for this step. Please find attached https://docs.google.com/document/d/1Rn72NMEo2QSErzoC6KPItp5bKjRwbqed8GWYDBqlHwg